lib/ansible/module_utils/common/respawn.py

```diff
@@ -55,11 +58,19 @@ def probe_interpreters_for_module(interpreter_paths, module_name):
     be returned (or ``None`` if probing fails for all supplied paths).
     :arg module_name: fully-qualified Python module name to probe for (eg, ``selinux``)
     """
+    env = os.environ | {'PYTHONPATH': _ANSIBLE_PARENT_PATH}
```

Comment:
If `PYTHONPATH` is already set, the existing value should be appended to instead of being overwritten.

Reply:
The goal in this iteration was to restrict importing the `ansible.module_utils.basic` for the interpreter test, to the specific file adjacent to respawn.  Without this, or at least prepending, we could import another install.

Reply:
Good point. Prepending would be safer then overwriting then, in case `module_name` should be found on `PYTHONPATH`. That would also be consistent with how AnsiballZ modifies `sys.path`.